clang/lib/Frontend/InitPreprocessor.cpp

```diff
@@ -754,6 +754,10 @@ static void InitializeCPlusPlusFeatureTestMacros(const LangOptions &LangOpts,
     Builder.defineMacro("__cpp_multidimensional_subscript", "202211L");
     Builder.defineMacro("__cpp_auto_cast", "202110L");
   }
+  // C++26 features.
+  if (LangOpts.CPlusPlus26) {
+    Builder.defineMacro("__cpp_variadic_friend", "202403L");
+  }
```

Comment:
I thought we already had an umbrella for C++26 features e.g. pack indexing expressions... @cor3ntin is it an oversight?

Reply:
We actually don’t define `__cpp_pack_indexing` yet, and the other C++26 features that we support that have a feature-test macro are also exposed in previous language modes, so this would be the first one that’s (currently) C++26 only (though, again, maybe we want to expose this one in older language modes too?).

I don’t actually know if this is the case, but I’m assuming the reason why we don’t define `__cpp_pack_indexing` is because there is a chance that the syntax might change or something like that?

Reply:
Update: `__cpp_pack_indexing` has since been added in a separate pr, but since we’re splitting this into multiple patches now, we won’t be adding a feature test macro for this just yet.